README.md

```diff
@@ -18,10 +19,15 @@ In a new container, you'll probably need the following:
 
 ```
 # Ubuntu 16
-apt-get install -y sudo git python
+apt-get update && apt-get install -y sudo git python
 
 # CentOS 7
-yum install -y sudo git python
+yum update && yum install -y sudo git python
+
+# AlmaLinux 8
+yum update && yum install -y sudo git python2 python2-virtualenv
+python2 -m virtualenv <some_directory>
+source <some_directory>/bin/activate
```

Comment:
AlmaLinux 8 does not provide a symlink for `python`. The solution I implemented was to simply introduce a virtualenv.

Thoughts?

Reply:
Agreed